### ISSUE TITLE
ERROR - Could not cancel stoploss order

### ISSUE THREAD

**Issue by dobremha** [freqtrade/freqtrade#7766]
Describe your environment
Operating system: Ubuntu 22.04 LTS
Python Version: 3.10.6
Freqtrade 2022.6
CCXT 1.89.14
Freqtrade running in docker : docker-compose up -d --remove-orphans
Echange: Binance

Hello everyone, i know this error is not new, but I can't understand the logic behind.
part from my strategy file:

    position_adjustment_enable = True

    stoploss = -0.33 
    trailing_stop = False

    order_types = {
        'entry': 'market',
        'exit': 'market',
        'stoploss': 'market',
        'stoploss_on_exchange': True
    }

When it tries to cancel a stop loss order it throws this error, but despite the error (when you click on the link in the error it will open {"code":-2014,"msg":"API-key format invalid."} message in browser), it does fine what he has to do (cancel the old SL order and put a new one width current stake), I have checked all my SL orders in Binance App.
It's happening ONLY when canceling SL orders, **after new DCA buy's**, to adjust SL for the new stake. Every time a DCA order is filled. 
All other functions/task's are working fine, no other error, so I don't see a API connection problem here. 
[err.txt](https://github.com/freqtrade/freqtrade/files/10047801/err.txt)

Even when a SL order cancel after ROI exit is happen, no error is thrown (see example). 
[err1.txt](https://github.com/freqtrade/freqtrade/files/10047803/err1.txt)

**Comment by xmatthias** [comment]
Please use the latest version if you encounter problems.
Many things have changed between 2022.6 and the current version  (2022.10).

It'd be a waste of time to look at issues that may have been solved either in freqtrade, or in ccxt months ago.

**Comment by dobremha** [comment]
ok, I have updated:
CCXT 2.0.58
Version=2022.10
Let's see next DCA orders after the update.

**Comment by dobremha** [comment]
Same error with new version.
[err2.txt](https://github.com/freqtrade/freqtrade/files/10049521/err2.txt)

**Comment by xmatthias** [comment]
Was this trade opened with the new version? (it doesn't seem to be - as the open-date was yesterday evening).

otherwise the error may have crept around from the outdated version.

**Comment by dobremha** [comment]
No, initial trade yesterday with old vers; but last DCA order from attachment with last version 2022.10.

**Comment by xmatthias** [comment]
well the logs list an order id - `883065083` as the open order.

you should be able to run the following query to fetch the order in question 

``` python
import ccxt
exchange = ccxt.binance({'apiKey': '<yourApiKey>', 'secret': '<yoursecret>' })
pair = 'KAVA/USDT'
res = exchange.fetch_order('883065083', pair)
print(res)
```

this will either succeed (or more likely, based on the logs) - fail - as the order ID is somehow wrong.
If it succeeds, it'll tell us the timing (when it was placed / cancelled).

In the end, this can/will most likely mean that the stoploss order ID was somehow messed up (it's stored when the order is created) - but that's a suspicion for now.

**Comment by dobremha** [comment]
Same with trade open after 2022.10 vers .... i try to import my last example
[err3.txt](https://github.com/freqtrade/freqtrade/files/10049612/err3.txt)

**Comment by xmatthias** [comment]
Please adjust the above script to the new trade and provide me with the output, thanks.

**edit**
how does the trade behave going forward?
after the exception, it seems to place a new stoploss order (one iteration later) - is that for the total amount (everything you own from this coin) ?

**Comment by dobremha** [comment]
Yes, new stoploss (one iteration later) is for my total coin amount which was bought in that trade (Initial buy + DCA1 + DCA2 etc.). In my opinion it's correct like this.
Result :
[err4.txt](https://github.com/freqtrade/freqtrade/files/10049871/err4.txt)

**Comment by xmatthias** [comment]
I've been able to reproduce this now. will be fixed soon.
